src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -44,7 +47,12 @@ def call(self):
         return self.sentry_app
 
     def _create_proxy_user(self):
-        return User.objects.create(username=self.name.lower(), is_sentry_app=True)
+        return User.objects.create(
+            username=u"{}-{}".format(
+                self.name.lower(), hashlib.sha1(self.organization.slug).hexdigest()[0:6]
```

Comment:
@nolachen Is `hashlib.sha1` going to generate a random UUID? That's what we really need here.